pipeline/relation_embedder/relation_embedder/src/main/scala/weco/pipeline/relation_embedder/lib/LambdaConfiguration.scala

```diff
@@ -0,0 +1,33 @@
+package weco.pipeline.relation_embedder.lib
+
+import java.io.File
+import com.typesafe.config.{Config, ConfigFactory}
+
+trait Configuration {
+  val config: Config
+}
+
+trait LambdaConfiguration extends Configuration {
```

Comment:
This kind of thing makes me happy.

Could we even go one step further and hide `val config` from the Lambda, instead translating the relevant config to a case class or trait so that all the string-based references to properties are in one place?

I suspect it has too many tendrils to do cleanly as part of this change, but might be a worthy ticket to raise for a future improvement